test/suite/test_import01.py

```diff
@@ -74,16 +74,18 @@ def check(self, uri, keys, values):
             else:
                 self.check_record(uri, keys[i], values[i])
 
-    # We know the ID can be different between configs, so just remove it from comparison.
-    # Everything else should be the same.
+    # The ID and checkpoint information can be different between configs, remove it. Everything else
+    # should be the same.
     def config_compare(self, aconf, bconf):
         a = re.sub('id=\d+,?', '', aconf)
```

Comment:
This isn't part of the current ticket, but the `id` regex will also match against the contents of the encryption field `encryption=(keyid=,` if the key starts with a digit. This won't impact correctness since the removal will happen on both configs